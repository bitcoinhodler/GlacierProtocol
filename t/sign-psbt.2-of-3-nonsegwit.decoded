{
  "txid": "f610cc3509ae64a9a0e38954b2d14fffd14b7ae6e688c9238727bb2c33aafd5a",
  "hash": "f610cc3509ae64a9a0e38954b2d14fffd14b7ae6e688c9238727bb2c33aafd5a",
  "version": 2,
  "size": 337,
  "vsize": 337,
  "weight": 1348,
  "locktime": 0,
  "vin": [
    {
      "txid": "26e7aa36be92259c8301afb4d50fed911a3d8654c2a795ade65762a9c51ba17f",
      "vout": 0,
      "scriptSig": {
        "asm": "0 304402203b49d38cb9f6f85956135eab53ad83b388ac2512c67efef7ee9e3e8d27dcc0ea02207e59438aa8e04d4916964e0b6deeba6c9628ab7389f9e39d69f655a00a6bb841[ALL] 3044022018fe977aab3b340c775885adc4daa9f6db6382b9e15fd1bf9521a14fa35f88b40220022b3febbed95c6415dcc84b0c82ddffc4739dd59600f111462ab5998dd487db[ALL] 52210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d653ae",
        "hex": "0047304402203b49d38cb9f6f85956135eab53ad83b388ac2512c67efef7ee9e3e8d27dcc0ea02207e59438aa8e04d4916964e0b6deeba6c9628ab7389f9e39d69f655a00a6bb84101473044022018fe977aab3b340c775885adc4daa9f6db6382b9e15fd1bf9521a14fa35f88b40220022b3febbed95c6415dcc84b0c82ddffc4739dd59600f111462ab5998dd487db014c6952210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d653ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.19996630,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 b6c7733e23a271e57ca16e5cd885174917968db9 OP_EQUALVERIFY OP_CHECKSIG",
        "desc": "addr(mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99)#fxf0ktmw",
        "hex": "76a914b6c7733e23a271e57ca16e5cd885174917968db988ac",
        "address": "mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99",
        "type": "pubkeyhash"
      }
    }
  ]
}
