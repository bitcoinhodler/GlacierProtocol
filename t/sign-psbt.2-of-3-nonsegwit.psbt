cHNidP8BAFUCAAAAAX+hG8WpYlfmrZWnwlSGPRqR7Q/VtK8Bg5wlkr42qucmAAAAAAD/////AdYfMQEAAAAAGXapFLbHcz4jonHlfKFuXNiFF0kXlo25iKwAAAAAAAEA/T8BAgAAAAFJbeX1LheP4/ooDkxrFoZsKrF0aP1vrxjnKQoigVApvgEAAABqRzBEAiAiGCnDUNl2VsCLF4pFVD6BbP05OAyIDo9X9TugrupjpAIgSCfv4RuVX649tg3cLOJw10nePObMnQ0RgR3s1c9jIm4BIQKsqN4LU0eBNvCBan8VINF7exggNNoq0mU8wb3RRg+rKf////8FAC0xAQAAAAAXqRT90gD24CB2FzKSZC/TUtxF+EkHDoeQQJcAAAAAABl2qRRIM3Jb+fHdRarOtVGavMU35y62zYisAC0xAQAAAAAXqRQks8J5X1QKXh/t4jg74FtLqpZ/1IcALTEBAAAAABepFMH1wcbIKsBzEQLMoDJ4EvVxLBc1hwAtMQEAAAAAF6kUk5JItkXWiUmy4GsSi+LiMLtRO6eHAAAAAAEEaVIhAyCElgfu0mXAn1HNoOlyQmVYJ2OEhwZRJQqk6GOMM0ztIQJ2bHBHKoo951HRJu3ayENjOPhEexlntao7EqK5pucS6SEDkIDdxEjn/kmFZ2p9LWVn38L+u4IHpvKblzcjP0w7RNZTrgAA
