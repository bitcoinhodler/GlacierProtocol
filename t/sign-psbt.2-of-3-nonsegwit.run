#!/bin/bash
set -e

$GLACIERSCRIPT --regtest=$1 sign-psbt << INPUT 2>&1
y
y
y
y
y
y
../../t/sign-psbt.2-of-3-nonsegwit.psbt
y
2
cVV1MBAgdAnrX8bjQWSJcgZAbVtprR8kgdfez858ejvXs3qZWqn2
cQy39MvkMKeaEvG1wXejdtEvAdU381u9EBBmcKUJWbfn6QL82YvR
y
INPUT
