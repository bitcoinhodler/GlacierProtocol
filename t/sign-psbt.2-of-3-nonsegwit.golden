Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.20000000 BTC in unspent inputs from cold storage address 2NGPJX8kzdRpAQJuZWMpCBth1umjQFHeFcz
0.19996630 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00003370
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 2 required to complete transaction)?
#: Key #1: Key #2: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.20000000 BTC in unspent inputs from cold storage address 2NGPJX8kzdRpAQJuZWMpCBth1umjQFHeFcz
0.19996630 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00003370

Signing with private keys: 
cVV1MBAgdAnrX8bjQWSJcgZAbVtprR8kgdfez858ejvXs3qZWqn2
cQy39MvkMKeaEvG1wXejdtEvAdU381u9EBBmcKUJWbfn6QL82YvR


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 10.00 satoshis per vbyte

Raw signed transaction (hex):
02000000017fa11bc5a96257e6ad95a7c254863d1a91ed0fd5b4af01839c2592be36aae72600000000fc0047304402203b49d38cb9f6f85956135eab53ad83b388ac2512c67efef7ee9e3e8d27dcc0ea02207e59438aa8e04d4916964e0b6deeba6c9628ab7389f9e39d69f655a00a6bb84101473044022018fe977aab3b340c775885adc4daa9f6db6382b9e15fd1bf9521a14fa35f88b40220022b3febbed95c6415dcc84b0c82ddffc4739dd59600f111462ab5998dd487db014c6952210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d653aeffffffff01d61f3101000000001976a914b6c7733e23a271e57ca16e5cd885174917968db988ac00000000

Transaction fingerprint (md5):
6c12563598aefb7616e529c2f94edfff
QR code for transaction written to transaction.png
